examples/07_automatic_preprocessing_with_the_supervectorizer.py

```diff
@@ -0,0 +1,178 @@
+    ],
+    axis=1,
+    inplace=True
+)
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=0.15, random_state=42
+)
+
+
+###############################################################################
+# Using the vectorizer
+# --------------------
+# Here is a simple workflow with the `SuperVectorizer`.
+
+from sklearn.ensemble import RandomForestRegressor
+from dirty_cat import SuperVectorizer
+
+sup_vec = SuperVectorizer(auto_cast=True)
+regressor = RandomForestRegressor(n_estimators=25, random_state=42)
+# Fit the SuperVectorizer
+X_train_enc = sup_vec.fit_transform(X_train, y_train)
+X_test_enc = sup_vec.transform(X_test)
+# And the regressor
+regressor.fit(X_train_enc, y_train)
+
+
+###############################################################################
+# Inspecting the features created
+# -------------------------------
+# Let's now break down what the `SuperVectorizer` did.
+#
+# Once it has been trained on data,
+# we can print the transformers and the columns assignment it creates:
+
+print(sup_vec.transformers_)
+
+###############################################################################
+# This is what is being passed to the |ColumnTransformer| under the hood.
+# If you're familiar with how the later works, it should be very intuitive.
+# We can notice it considered the columns "gender" and "assignment_category"
+# as low cardinality string variables.
+#
+# The vectorizer actually makes the difference between string variables
+# (data type `object` and `string`) and categorical variables
+# (data type `category`).
+# A |OneHotEncoder| will be applied to these columns.
+#
+# Next, we can have a look at the encoded feature names.
+
+# Before:
+print(X.columns.to_list())
+# After :
+feature_names = sup_vec.get_feature_names()
+print(', '.join(feature_names[:10]), '...')
+print(len(feature_names))
+
+###############################################################################
+# As we can see, it created a new column for each unique value.
+# This is because we used |SimilarityEncoder| on the column "division",
+# which was classified as a high cardinality string variable.
+# (default values, see `SuperVectorizer`'s docstring).
+# In total, we have 1212 encoded columns.
+#
+# Finally, let's plot the feature importances.
+#
+# .. topic:: Note:
+#
+#    we will plot the feature importances computed by the
+#    |RandomForestRegressor|, but you should use |permutation importances|
+#    instead (which are much more accurate)
+#
+#    We chose the former over the later for the sake of performance.
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Getting feature importances
+importances = regressor.feature_importances_
+std = np.std(
+    [
+        tree.feature_importances_
+        for tree in regressor.estimators_
+    ],
+    axis=0
+)
+indices = np.argsort(importances)[::-1]
+
+# Plotting the results:
+
+plt.figure(figsize=(18, 9))
```

Comment:
```suggestion
plt.figure(figsize=(12, 9))
```
Make the figure more narrow, so that the text are more visible